src/api/handlers/TutorInsertHandler.ts

```diff
@@ -1,21 +1,22 @@
-import { Handler, ProxyResult } from "aws-lambda";
+import { APIGatewayEvent, Handler, ProxyResult } from "aws-lambda";
+import { DatabaseServerlessHandler } from "../core/DatabaseServerlessHandler";
 import { ProxyResultBuilder } from "../core/ProxyResultBuilder";
 import { TutorService } from '../services/TutorService';
 
-class TutorInsertHandler {
-    private tutorService: TutorService;
+class TutorInsertHandler extends DatabaseServerlessHandler<APIGatewayEvent> {
+    private tutorService: TutorService | undefined;
 
-    constructor() {
-        this.tutorService = new TutorService();
+    initializeDependencies(): void {
+        this.tutorService = new TutorService(this.connection);
     }
 
-    public execute(event: any): ProxyResult {
+    public async onHandleEvent(event: APIGatewayEvent): Promise<ProxyResult> {
         const response = this.tutorService.insert(JSON.parse(event.body));        
```

Comment:
@Aristevao about [this comment](https://github.com/Aristevao/serverlessoffline/pull/16#issuecomment-879345374): 
- Here you need to see the code of the `insert()` function of the `tutor` service.
- The function `insert()` is an `async` function?

Reply:
@bigLucas 
Yes, the `insert()` function from `tutor` service is an `async` function.
Your point is that the `await` keyword is missing before `this.tutorService`, right?

Reply:
@Aristevao you can make a `commit` with this fix, remember to fix all handler files.

Reply:
Done!